### ISSUE TITLE
Scaling (via diagrams) doesn't work

### ISSUE THREAD

**Issue by afcady** [HeinrichApfelmus/hyper-haskell#40]
Apparently there is some kind of autoscale in the rendering process, which sometimes makes for absurd results:

```haskell
-- Should fit on screen without scrolling, but does not.
dia $ circle 1 # scaleY 100 # scale 0.01
```

In general, any `scale` applied to the entire diagram has no effect.

It also makes it so that a diagram does not maintain a constant size after operations that add other diagrams next to it.  So:

```haskell
-- Now it fits on the page!  But adding a circle on the
-- right shouldn't change the size of anything on the left.
dia $ circle 1 # scaleY 100 # scale 0.01 ||| circle 1
```

**Comment by HeinrichApfelmus** [comment]
Yes, sorry about that. The function `dia` chooses a particular width for rendering — `250` (in pixels, I believe). I'm not sure what a good default choice would be — maybe make it a parameter?

**Comment by HeinrichApfelmus** [comment]
After some experimentation, I'm settling on the absolute size of "one diagram unit = `1px`". This gives sizes that are predictable.
